source/MaterialXGenShader/GenOptions.cpp

```diff
@@ -14,7 +14,8 @@ GenOptions::GenOptions() :
     hwTransparency(false),
     hwSpecularEnvironmentMethod(SPECULAR_ENVIRONMENT_FIS),
     hwMaxActiveLightSources(3),
-    hwNormalizeUdimTexCoords(false)
+    hwNormalizeUdimTexCoords(false),
+    textureSpaceRender(false)
```

Comment:
If a new derived shader generator was used for texture baking we would not need an option for this.